Makefile

```diff
@@ -1,4 +1,5 @@
 # Makefile for the documentation
+SHELL := /bin/bash
```

Comment:
I guess the shell variable doesn't really matter :)

Reply:
No, you're right. That was a leftover from when I was stuck. I'll remove it.